distribution/bin/enso.bat

```diff
@@ -1,3 +1,10 @@
-@set comp-dir=%~dp0\..\component
-@java -jar -Dtruffle.class.path.append=%comp-dir%\runtime.jar -Dpolyglot.engine.IterativePartialEscape=true %JAVA_OPTS% %comp-dir%\runner.jar %*
-@exit /B %errorlevel%
+@echo off
+set comp-dir=%~dp0\..\component
+set EXTRA_OPTS="Dgraal.PrintGraph=Network
+FOR %%A in (%*) DO (
```

Comment:
Looks like I broke the [windows build](https://github.com/enso-org/enso/runs/7006741374?check_suite_focus=true) with this change.

I found this [trick on web](https://serverfault.com/questions/118272/checking-parameters-in-a-dos-batch-file) but I have limited chances to test it (being a Linux user). Any help from _Windows users_ welcomed. CCing @mwu-tow , @jdunkerley & co.

Reply:
Probably fixed by [1aab449](https://github.com/enso-org/enso/pull/3533/commits/1aab4490fa48411e844a25a166518035f6b0046a)